docs/current_docs/api/module-structure.mdx

```diff
@@ -267,7 +267,7 @@ This can be useful to add a few requirements to the module's execution environme
 </TabItem>
 <TabItem value="TypeScript">
 
-The runtime container is currently hardcoded to run in Node.js 21.3 (although this may be configurable in future).
+The runtime container is currently hardcoded to run in Node.js 22.11.0 (although this may be configurable in future).
```

Comment:
It's already configurable so this is outdated.

Reply:
Fixed